.github/README_fr.md

```diff
@@ -21,7 +21,7 @@
         <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/by.svg">
     </a>
     <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_fr.md">
-        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/fr.svg">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/fr.svg">
```

Comment:
The selected flag should get larger (25px) when the user navigates to it. In this case the French flag should be 25px.
```suggestion
        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/fr.svg">
```